src/bridge/mod.rs

```diff
@@ -98,11 +98,12 @@ impl<IS: AsyncRead + AsyncWrite + 'static + Send> Bridge<IS> {
             IrcCommand::PrivMsg { channel, text } => {
                 if let Some(room_id) = self.mappings.channel_to_room_id(&channel) {
                     info!(self.ctx.logger, "Got msg"; "channel" => channel.as_str(), "room_id" => room_id.as_str());
-                    self.matrix_client
+
+                    let _ = self
```

Comment:
Oh, sorry, I guess I misread this. If we actually do something on `Err` (run `task_warn` in this case), that shouldn't be a `map_err` at all, instead you can use `if self.matrix_client.send_text_message(room_id, text).await.is_err() { task_warn!("Failed to send"); }`.

Reply:
I agree, updated it